HelpSource/Classes/ArrayedCollection.schelp

```diff
@@ -465,14 +465,16 @@ a.reshape(2,3);
 ::
 
 method::find
-Finds the starting index of a number of elements contained in the array.
+Finds the starting index of a number of elements contained in the array. This method expects a collection as an argument. So for finding only one element, place the element inside a collection.
```

Comment:
Wouldn't it be better to recommend `indexOfEqual` when one searches only for one element? At least it is more efficient, I think.

Reply:
Oh, yes thats a very good point. I did not think about this.